ciri/api/BUILD

```diff
@@ -5,6 +5,8 @@ cc_library(
     srcs = ["api.c"],
     hdrs = ["api.h"],
     deps = [
+        "//cclient/request:requests",
```

Comment:
should we move `request/response` to a neutral namespace and use it as a dependency on both cIRI/cclient ?

Reply:
I think so, and serialization too.
Any suggestion where ?

Reply:
`common/network`  ?

Reply:
For the record: we decided to hold off on moving this for now